autogen.sh

```diff
@@ -11,3 +11,5 @@ autoreconf --verbose --force --install || exit $?
 
 cd "$cwd"
 test -z "$NOCONFIGURE" && "$srcdir/configure" $@
+
+gtkdocize || exit 1
```

Comment:
This needs to be done above right below the `autoreconf` command. Right now, this assumes that `autogen.sh` is run from the source directory via `./autogen.sh`.